.github/reviewer-lottery.yml

```diff
@@ -0,0 +1,44 @@
+groups:
+  # Default reviewers for all pull requests.
+  # Usually, at least on of the maintainers should approve PR before merging.
+  # The best is if two maintainers do that.
+  - name: maintainers # name of the group
+    reviewers: 2 # how many reviewers do you want to assign?
+    internal_reviewers: 1 # how many reviewers do you want to assign when the PR author belongs to this group?
+    usernames: # github usernames of the reviewers
+      - bmagyar
+      - destogl
+
+  # Reviewers group to get broader feedback.
+  - name: reviewers
+    reviewers: 5
+    usernames:
+      - rosterloh
+      - kellyprankin
+      - progtologist
+      - arne48
+      - DasRoteSkelett
+      - a10263790
+      - harderthan
+      - jaron-l
+      - malapatiravi
+      - erickisos
+      - ShawnSchaerer
+      - TomoyaFujita2016
+      - homalozoa
+      - anfemosa
+      - jackcenter
+      - VX792
+      - mhubii
+      - livanov93
+      - aprotyas
+      - peterdavidfagan
+      - UsamaHamayun1
+      - bijoua29
+      - Aposhian
+      - safdark
+      - rk800506
+      - chrismath
+      - lauesa
+      - gokulmaxi
+      - Briancbn
```

Comment:
```suggestion
      - rosterloh
      - kellyprankin
      - progtologist
      - arne48
      - DasRoteSkelett
      - a10263790
      - Serafadam
      - harderthan
      - jaron-l
      - malapatiravi
      - erickisos
      - ShawnSchaerer
      - Briancbn
      - TomoyaFujita2016
      - homalozoa
      - anfemosa
      - jackcenter
      - VX792
      - mhubii
      - livanov93
      - aprotyas
      - peterdavidfagan
      - UsamaHamayun1
      - duringhof
      - bijoua29
      - kasiceo
```

Reply:
updated with the list of reviewers as per this morning (not adding the ones who were only invited but didn't accept)